packages/flyout/src/Flyout.js

```diff
@@ -289,7 +291,7 @@ export default class Flyout extends Component {
 
     if (React.Children.count(children) === 1) {
       return React.cloneElement(children, {
-        onClick: handleChildClick,
+        onClick: combineEventHandlers(handleChildClick, children.props.onClick),
         onMouseEnter,
```

Comment:
Right now seems using the `utils.combineEventHandlers` to combine the origin `Button`'s onClick prop and the Flyout's clickHandler function will get working as expected. Now sure if we need make similar combines for onMouseEnter/Leave ?

Reply:
From what I can tell this is working as it should, as now in the `Tooltip` component when I have `openOnHover` it looks like the the child `onClick` is going through. 

I don't thinke we need for onMouseEnter/Leave and can be merged after rebasing.